modules/apps/29-fee/client/cli/tx.go

```diff
@@ -28,7 +28,7 @@ func NewPayPacketFeeAsyncTxCmd() *cobra.Command {
 		Use:     "pay-packet-fee [src-port] [src-channel] [sequence]",
 		Short:   "Pay a fee to incentivize an existing IBC packet",
 		Long:    strings.TrimSpace(`Pay a fee to incentivize an existing IBC packet.`),
-		Example: fmt.Sprintf("%s tx pay-packet-fee transfer channel-0 1 --recv-fee 10stake --ack-fee 10stake --timeout-fee 10stake", version.AppName),
+		Example: fmt.Sprintf("%s tx ibc-fee pay-packet-fee transfer channel-0 1 --recv-fee 10stake --ack-fee 10stake --timeout-fee 10stake", version.AppName),
```

Comment:
How do you feel about doing instead `tx ibc fee`? We have under `tx` currently `ibc` and `ibc-transfer`, should we try to group things under the `ibc` subcommand? Maybe this requires some discussion to think about the direction we want to go regarding cli commands and if refactoring would be needed for the commands we currently have.

Reply:
Thanks for raising this point. I think it's worthy of discussion. I think it's worth merging in this PR as-is for now and following up.

Reply:
Might be difficult to do since then certain applications maintained by us can be in `ibc` subcommand. But third-party ibc applications won't be. This will make it confusing if those apps are in same binary, since it won't be clear why some ibc apps get to be in `ibc` subcommand and others aren't

Reply:
Thanks @AdityaSripal. Yes, @colin-axner also shared the same concern with me. I will just open an issue for now to work on it in the future. So as @seantking suggests it's good to merge as is.